ocs_ci/deployment/deployment.py

```diff
@@ -247,3 +241,24 @@ def add_node(self):
         Implement platform specif add_node in child class
         """
         raise NotImplementedError("add node functionality node implemented")
+
+    def patch_default_sc_to_non_default(self):
+        """
+        Patch storage class which comes as default with installation to non-default
+        """
+        sc_to_patch = None
+        platform = config.ENV_DATA.get('platform')
+        if platform in constants.AWS_PLATFORM:
+            sc_to_patch = constants.DEFAULT_SC_AWS
+        elif platform in constants.VSPHERE_PLATFORM:
+            sc_to_patch = constants.DEFAULT_SC_VSPHERE
+        else:
+            logger.info(f"Unsupported platform {platform} to patch")
+        if sc_to_patch:
```

Comment:
If you will follow above mentioned you can have just if self.default_sc or self.sc_to_patch and ommit the code above?

Reply:
same as above comment